charts/gitops-runtime/templates/hooks/pre-install/validate-usage.yaml

```diff
@@ -0,0 +1,49 @@
+{{- if  not .Values.installer.skipUsageValidation }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: validate-values-config
+  annotations:
+    helm.sh/hook: pre-install
+    helm.sh/hook-delete-policy: hook-succeeded,before-hook-creation,hook-failed
+    helm.sh/hook-weight: "-10"
```

Comment:
It has the same weight as `validate-values`, I think it should run after the `validate-values`. Let's adjust the weight.